.github/actions/install-dashboards/action.yml

```diff
@@ -53,7 +50,7 @@ runs:
       shell: bash
 
     - id: branch-switch-if-possible
-      continue-on-error: true # Defaults onto main if the branch switch doesn't work
+      continue-on-error: true 
```

Comment:
Do you know why the branch switch would not work or if there is anything we can do to make sure it is successful?

Reply:
Seems like there should still be a comment to justify this non-standard behavior what do you think about keeping the comment or would it be better to clarify it?